src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBERS_BOUND = 100;
+    private Ball randomBall = new Ball();

```

Comment:
do you really need this class field?

Reply:
Yes, you're right. It isn't neccesary.
For working three different Balls-creation in class Application i will relocate <Ball randomBall = new Ball()> to the method getRandomBall().
This statement <private static final int NUMBERS_BOUND = 100;> i used according to rules "All magic numbers in your code should be constants" 
Maybe it isn't neccesary to be private?